src/main/java/org/broadinstitute/barclay/help/DefaultDocWorkUnitHandler.java

```diff
@@ -602,11 +614,16 @@ private FieldDoc getFieldDoc(final ClassDoc classDoc, final String name, final b
             if (fieldDoc.name().equals(name))
                 return fieldDoc;
 
+            // this can return null if the fieldDoc represents a field that has javadoc, but is private.
             final Field field = DocletUtils.getFieldForFieldDoc(fieldDoc);
-            if (field == null)
-                throw new RuntimeException("Could not find the field corresponding to " + fieldDoc +
-                        ", presumably because the field is inaccessible");
-            if (field.isAnnotationPresent(ArgumentCollection.class)) {
+            if (field == null) {
+                logger.warn(
```

Comment:
This code path is already trying to determine if the field is an ArgumentCollection, but it can't do that if it can't retrieve the Field. The problem with throwing here is that it aborts the whole run, even when it would succeed if it just kept searching. We only want to throw when we've exhausted the search and can't find the fieldDoc for an argument at all, which happens later on in the process.